woodpecker-server/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: woodpecker-server
 description: A Helm chart for the Woodpecker server
 type: application
-version: <version>
-appVersion: "v<version>"
+version: 1.0.0
```

Comment:
As I understand `<version>` is replaced with real value by the release pipeline: https://github.com/woodpecker-ci/helm/blob/c319fca9ea79f90a3612561904f75c6f97c19168/.woodpecker.yml#L13 

So it will break the automated release process.

I'm guessing the main reason for this change is to enable installing from git. One way to enable it is to commit to manually updating the chart version every time when something changes. But it is quite a change in process and requires automation IMHO (e.g. run https://github.com/helm/chart-testing during CI to make sure version is updated on each chart change)

Reply:
Thanks for pointing that out! Didn't know that.

Having a placeholder for version in the main branch of a helm chart is quite unusual I'd say. Normally user don't need to install a chart from git, yes - but especially because of the drift between the latest release and "next" in WP this is really important right now.

I think there must be a better way to handle the release process than using this placeholder here?

Reply:
In https://github.com/argoproj/argo-helm we've decided to just bump the version every time when helm chart is changed.  The lint step that uses https://github.com/helm/chart-testing ensures we don't forget to do it. This process worked pretty well for us.

So my recommendation would be to remove replacement from release pipeline and optionally add chart limiting. Need opinion from one of the maintainers though :)